explore_utils.py

```diff
@@ -101,3 +101,21 @@ def find_channel_grouping_revenue(dataset):
     means = {grouping: np.mean(df[df['channelGrouping'] == grouping]['totals.transactionRevenue'].astype('int64')) / 10000 for grouping in groupings}
 
     return counts, means
+
+def find_transaction_by_region(data):
+    """ Find the average transaction revenue by region
+
+    args:
+        dataset (Dataset): the google analytics Dataset
+
+    returns:
+        Dataframe of total transaction revenue by region in ascending order.
+
+    """
+    train_df = data.train
+    train_df['rev'] = train_df['totals.transactionRevenue'].fillna(0).astype(float)
+    idx = train_df.groupby('geoNetwork.region')['rev'].mean()
```

Comment:
(Optional) I think `idx` is somewhat of a confusing name here, this typically is used to mark a Series that is used to index some dataframe. In this case, `idx` refers to the data, not the index.